docs/modules/quickstarters/pages/index.adoc

```diff
@@ -32,7 +32,7 @@ This repository contains quickstarters, which are basically templates that help
 === Other Quickstarter
 * xref::e2e-cypress.adoc[E2E test - Cypress]
 
-Next to those, there is a https://github.com/opendevstack/ods-quickstarters/blob/master/boilerplates/docker-plain/README.md["blank" quickstarter] allowing to start from scratch, while still providing all the OCP resources / Jenkins / SonarQube conveniences described in the following sections.
+Next to those, there is a https://github.com/opendevstack/ods-quickstarters/blob/2.x/README.md["blank" quickstarter] allowing to start from scratch, while still providing all the OCP resources / Jenkins / SonarQube conveniences described in the following sections.
```

Comment:
This should actually be an `xref`, like above: `xref::e2e-cypress.adoc[E2E test - Cypress]`

Reply:
@ggoek - do you address this? ...

Reply:
@clemensutschig - Hi I just changed it to xref. I hope it's right?